tests/dev/test-raltegravir-nonparallel-population.R

```diff
@@ -23,15 +23,14 @@ popWorkFlow <- PopulationWorkflow$new(simulationSets = list(RaltegravirSimSet, L
 
 
 popWorkFlow$simulatePopulation$settings$showProgress <- TRUE
-
-
-popWorkFlow$populationSensitivityAnalysis$settings$variableParameterPaths <- simTree1$Organism$Heart$Volume$path
-
-
-popWorkFlow$populationSensitivityAnalysis$settings$pkParameterSelection <- c("C_max")
-
-
-popWorkFlow$populationSensitivityAnalysis$settings$quantileVec <- c(0.5)
+popWorkFlow$simulatePopulation$settings$numberOfCores <- 1
+
+popWorkFlow$populationSensitivityAnalysis$settings$showProgress <- FALSE
```

Comment:
I'd suggest to create a variable settings <- popWorkFlow$populationSensitivityAnalysis$settings and set all the properties on the object instead of repeating `popWorkFlow$populationSensitivityAnalysis$settings` n times. 
People need to understand the beauty of working with reference